webapp/src/components/dropdown/index.tsx

```diff
@@ -5,7 +5,7 @@ import './styles.scss';
 type DropdownProps = {
     value: string | null;
     placeholder: string;
-    onChange: (newValue: string, targetLabel?: string) => void;
+    onChange: (newValue: string, targetLabel?: string, selectedOption?: Record<string, string>) => void;
```

Comment:
no because the parent component where we will use this `Dropdown` will require to pass this `prop` as `LabelValuePair` only but sometimes we need to give a different object type whose type varies